modules/localization/ndt/ndt_locator/BUILD

```diff
@@ -52,7 +52,7 @@ cc_test(
 
 cc_test(
     name = "ndt_solver_test",
-    size = "small",
+    size = "enormous",
     timeout = "short",
```

Comment:
So, `size="large"` is not enough ?
See https://docs.bazel.build/versions/master/be/common-definitions.html for detailed comparison of TEST_SIZE and TIMEOUT.

Reply:
since source is not a limit, why not make test time shorter?

Reply:
> since source is not a limit, why not make test time shorter?

It's generally considered good practice to be light-weight if possible rather than heavy-weight.